betty/_package/pyinstaller/__init__.py

```diff
@@ -52,12 +50,16 @@ async def a_pyz_exe_coll() -> tuple[Analysis, PYZ, EXE, COLLECT]:
     else:
         raise RuntimeError(f"Unsupported platform {sys.platform}.")
 
-    await _build_assets()
+    await prebuild_webpack_assets()
     block_cipher = None
     datas = []
     data_file_path_patterns = [
+        ".browserslistrc",
```

Comment:
Group these, and label the groups with comments, explaining why those files are needed